apps/frontend/static_src/scss/mixins.scss

```
@@ -0,0 +1,35 @@
+@use "sass:list";
+
+// Media queries
+@mixin media-query($queries...) {
+    @each $query in $queries {
+        @each $breakpoint in $breakpoints {
+            $name: list.nth($breakpoint, 1);
+            $declaration: list.nth($breakpoint, 2);
+
+            @if $query == $name and $declaration {
+                @media only screen and #{$declaration} {
+                    @content;
+                }
+            }
+        }
+    }
+}
+
+// Layout
+@mixin grid-layout() {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    padding: 0 $gutter;
+    max-width: $site-width;
+    margin: 0 auto;
+
+    @include media-query(medium) {
+        padding: 0 ($gutter * 3);
+    }
+
+    @include media-query(large) {
+        grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```

Just a tad easier to read

```suggestion
        grid-template-columns: repeat(10, 1fr);
```